test/tests/meshgenerators/sidesets_bounding_box_generator/deprecated_param.i

```diff
@@ -0,0 +1,24 @@
+[Mesh]
```

Comment:
no need to test deprecated parameters btw. We ve kind of made that call to gain time.